### PR TITLE
Improve worker config, refs #13109

### PR DESCRIPTION
**Pull request by jraddaoui**

### CODE REVIEW

File: admin-manual/installation/asynchronous-jobs.rst [hunk 1]
```diff
@@ -110,16 +115,35 @@ You can have access to the journal of our new ``atom-worker`` unit as follows:
...
+in an application setting under :file:`config/gearman.yml` and defaults to
+``127.0.0.1:4730``.
 
 Note that the job server will perfectly handle multiple workers running
 simultaneously and the work load will be distributed across all available
 workers. If there are no workers available because they are busy completing
 other tasks, the job server will store the job in the queues and deliver them
 once a worker becomes available.
+
+If your planing to connect multiple AtoM instances to the same Gearman server,
```
Comment (fiver-watson) [inline admin-manual/installation/asynchronous-jobs.rst]: you're

File: admin-manual/installation/upgrading.rst [hunk 2]
```diff
@@ -241,6 +241,14 @@ See :ref:`maintenance-clear-cache` for more detailed instructions.
 
 .. _upgrading-restart-services:
 
+Check the services configuration
+================================
+
+It's not usual, but sometimes we introduce changes in the related services
+configuration, like PHP-FPM, MySQL or the AtoM worker, to improve the behaviour
+and performance of the application. Check the installation documentaion for your
```
Comment (fiver-watson) [inline admin-manual/installation/upgrading.rst]: Minor typo - documentation

**Review by fiver-watson — CHANGES_REQUESTED** [review]
2 very minor typos to fix - otherwise it looks good!

**Review by fiver-watson — APPROVED** [review]
Fixes look good! Might want to squash before merging but otherwise 👍

**Comment by jraddaoui** [comment]
Thanks @fiver-watson!

File: admin-manual/installation/asynchronous-jobs.rst [hunk 3]
```diff
@@ -110,16 +115,35 @@ You can have access to the journal of our new ``atom-worker`` unit as follows:
...
+in an application setting under :file:`config/gearman.yml` and defaults to
+``127.0.0.1:4730``.
 
 Note that the job server will perfectly handle multiple workers running
 simultaneously and the work load will be distributed across all available
 workers. If there are no workers available because they are busy completing
 other tasks, the job server will store the job in the queues and deliver them
 once a worker becomes available.
+
+If you're planing to connect multiple AtoM instances to the same Gearman server,
```
Comment (djjuhasz) [inline admin-manual/installation/asynchronous-jobs.rst:142]: Typo, should be "planning"

**Pull request merged**